### PR TITLE
Add kyverno resources for etcd quota error

### PR DESCRIPTION
**Pull request by rverchere**
- Add reportchangerequest.kyverno.io as a resource that can increase very fast on a cluster
- Specify podvolumebackups.velero.io instead of podvolumebackups to be more accurate
- Add missing canonical routes on some pages

### CODE REVIEW

File: pages/platform/kubernetes-k8s/etcd-quota-error/guide.de-de.md [hunk 1]
```diff
@@ -141,10 +147,10 @@ kubectl get ingress.extensions -A | wc -l
 kubectl get authrequests.dex.coreos.com -A | wc -l
 ```
 
-- `podvolumebackups`
+- `reportchangerequest.kyverno.io
```
Comment (scraly) [inline pages/platform/kubernetes-k8s/etcd-quota-error/guide.de-de.md]:
```suggestion
- `reportchangerequest.kyverno.io`
```

File: pages/platform/kubernetes-k8s/etcd-quota-error/guide.en-asia.md [hunk 2]
```diff
@@ -139,10 +147,10 @@ kubectl get ingress.extensions -A | wc -l
 kubectl get authrequests.dex.coreos.com -A | wc -l
 ```
 
-- `podvolumebackups`
+- `reportchangerequest.kyverno.io
```
Comment (scraly) [inline pages/platform/kubernetes-k8s/etcd-quota-error/guide.en-asia.md]:
```suggestion
- `reportchangerequest.kyverno.io`
```

File: pages/platform/kubernetes-k8s/etcd-quota-error/guide.en-au.md [hunk 3]
```diff
@@ -139,10 +147,10 @@ kubectl get ingress.extensions -A | wc -l
 kubectl get authrequests.dex.coreos.com -A | wc -l
 ```
 
-- `podvolumebackups`
+- `reportchangerequest.kyverno.io
```
Comment (scraly) [inline pages/platform/kubernetes-k8s/etcd-quota-error/guide.en-au.md]:
```suggestion
- `reportchangerequest.kyverno.io`
```

File: pages/platform/kubernetes-k8s/etcd-quota-error/guide.en-gb.md [hunk 4]
```diff
@@ -139,10 +145,10 @@ kubectl get ingress.extensions -A | wc -l
 kubectl get authrequests.dex.coreos.com -A | wc -l
 ```
 
-- `podvolumebackups`
+- `reportchangerequest.kyverno.io
```
Comment (scraly) [inline pages/platform/kubernetes-k8s/etcd-quota-error/guide.en-gb.md]:
```suggestion
- `reportchangerequest.kyverno.io`
```

File: pages/platform/kubernetes-k8s/etcd-quota-error/guide.en-ie.md [hunk 5]
```diff
@@ -139,10 +147,10 @@ kubectl get ingress.extensions -A | wc -l
 kubectl get authrequests.dex.coreos.com -A | wc -l
 ```
 
-- `podvolumebackups`
+- `reportchangerequest.kyverno.io
```
Comment (scraly) [inline pages/platform/kubernetes-k8s/etcd-quota-error/guide.en-ie.md]:
```suggestion
- `reportchangerequest.kyverno.io`
```

File: pages/platform/kubernetes-k8s/etcd-quota-error/guide.en-sg.md [hunk 6]
```diff
@@ -139,10 +147,10 @@ kubectl get ingress.extensions -A | wc -l
 kubectl get authrequests.dex.coreos.com -A | wc -l
 ```
 
-- `podvolumebackups`
+- `reportchangerequest.kyverno.io
```
Comment (scraly) [inline pages/platform/kubernetes-k8s/etcd-quota-error/guide.en-sg.md]:
```suggestion
- `reportchangerequest.kyverno.io`
```

File: pages/platform/kubernetes-k8s/etcd-quota-error/guide.es-us.md [hunk 7]
```diff
@@ -141,10 +147,10 @@ kubectl get ingress.extensions -A | wc -l
 kubectl get authrequests.dex.coreos.com -A | wc -l
 ```
 
-- `podvolumebackups`
+- `reportchangerequest.kyverno.io
```
Comment (scraly) [inline pages/platform/kubernetes-k8s/etcd-quota-error/guide.es-us.md]:
```suggestion
- `reportchangerequest.kyverno.io`
```

File: pages/platform/kubernetes-k8s/etcd-quota-error/guide.en-ca.md [hunk 8]
```diff
@@ -139,10 +147,10 @@ kubectl get ingress.extensions -A | wc -l
 kubectl get authrequests.dex.coreos.com -A | wc -l
 ```
 
-- `podvolumebackups`
+- `reportchangerequest.kyverno.io
```
Comment (scraly) [inline pages/platform/kubernetes-k8s/etcd-quota-error/guide.en-ca.md]:
```suggestion
- `reportchangerequest.kyverno.io`
```

File: pages/platform/kubernetes-k8s/etcd-quota-error/guide.en-us.md [hunk 9]
```diff
@@ -139,10 +147,10 @@ kubectl get ingress.extensions -A | wc -l
 kubectl get authrequests.dex.coreos.com -A | wc -l
 ```
 
-- `podvolumebackups`
+- `reportchangerequest.kyverno.io
```
Comment (scraly) [inline pages/platform/kubernetes-k8s/etcd-quota-error/guide.en-us.md]:
```suggestion
- `reportchangerequest.kyverno.io`
```

File: pages/platform/kubernetes-k8s/etcd-quota-error/guide.es-es.md [hunk 10]
```diff
@@ -141,10 +147,10 @@ kubectl get ingress.extensions -A | wc -l
 kubectl get authrequests.dex.coreos.com -A | wc -l
 ```
 
-- `podvolumebackups`
+- `reportchangerequest.kyverno.io
```
Comment (scraly) [inline pages/platform/kubernetes-k8s/etcd-quota-error/guide.es-es.md]:
```suggestion
- `reportchangerequest.kyverno.io`
```

File: pages/platform/kubernetes-k8s/etcd-quota-error/guide.fr-ca.md [hunk 11]
```diff
@@ -141,10 +147,10 @@ kubectl get ingress.extensions -A | wc -l
 kubectl get authrequests.dex.coreos.com -A | wc -l
 ```
 
-- `podvolumebackups`
+- `reportchangerequest.kyverno.io
```
Comment (scraly) [inline pages/platform/kubernetes-k8s/etcd-quota-error/guide.fr-ca.md]:
```suggestion
- `reportchangerequest.kyverno.io`
```

File: pages/platform/kubernetes-k8s/etcd-quota-error/guide.fr-fr.md [hunk 12]
```diff
@@ -141,10 +147,10 @@ kubectl get ingress.extensions -A | wc -l
 kubectl get authrequests.dex.coreos.com -A | wc -l
 ```
 
-- `podvolumebackups`
+- `reportchangerequest.kyverno.io
```
Comment (scraly) [inline pages/platform/kubernetes-k8s/etcd-quota-error/guide.fr-fr.md]:
```suggestion
- `reportchangerequest.kyverno.io`
```

File: pages/platform/kubernetes-k8s/etcd-quota-error/guide.it-it.md [hunk 13]
```diff
@@ -141,10 +147,10 @@ kubectl get ingress.extensions -A | wc -l
 kubectl get authrequests.dex.coreos.com -A | wc -l
 ```
 
-- `podvolumebackups`
+- `reportchangerequest.kyverno.io
```
Comment (scraly) [inline pages/platform/kubernetes-k8s/etcd-quota-error/guide.it-it.md]:
```suggestion
- `reportchangerequest.kyverno.io`
```

File: pages/platform/kubernetes-k8s/etcd-quota-error/guide.pl-pl.md [hunk 14]
```diff
@@ -141,10 +147,10 @@ kubectl get ingress.extensions -A | wc -l
 kubectl get authrequests.dex.coreos.com -A | wc -l
 ```
 
-- `podvolumebackups`
+- `reportchangerequest.kyverno.io
```
Comment (scraly) [inline pages/platform/kubernetes-k8s/etcd-quota-error/guide.pl-pl.md]:
```suggestion
- `reportchangerequest.kyverno.io`
```

File: pages/platform/kubernetes-k8s/etcd-quota-error/guide.pt-pt.md [hunk 15]
```diff
@@ -141,10 +147,10 @@ kubectl get ingress.extensions -A | wc -l
 kubectl get authrequests.dex.coreos.com -A | wc -l
 ```
 
-- `podvolumebackups`
+- `reportchangerequest.kyverno.io
```
Comment (scraly) [inline pages/platform/kubernetes-k8s/etcd-quota-error/guide.pt-pt.md]:
```suggestion
- `reportchangerequest.kyverno.io`
```

**Review by scraly — CHANGES_REQUESTED** [review]
Thanks Remi! :-) 
Just a little missing backtick :-)

**Comment by rverchere** [comment]
> Thanks Remi! :-) Just a little missing backtick :-)

Oh! Oops, wrong copy-paste, sorry!

**Review by scraly — APPROVED** [review]
Thx! :)

**Review by bs-ovh — APPROVED** [review]

**Comment by bs-ovh** [comment]
Thank you @rverchere !
Please do not add canonicals to EN subs.

**Pull request merged**